Update index.md (#3043)
vega/vega: docs/usage/index.md

</head>
```

> **Using Vega with a bundler.** If using Vega with a package manager (e.g. [npm](https://www.npmjs.com/package/vega)) and a bundler like [rollup.js](https://rollupjs.org/guide/en/#with-npm-packages), you can import everything into a namespace as follows:

Reviewer: ```suggestion
**Using Vega with a bundler.** If you use Vega with a bundler like [rollup.js](https://rollupjs.org/guide/en/#with-npm-packages), you can import Vega [as a module](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import).
```
